CONTRIBUTING.md

```diff
@@ -4,6 +4,6 @@
 
 ```bash
 rockcraft pack -v
-sudo skopeo --insecure-policy copy oci-archive:vault_1.14.3_amd64.rock docker-daemon:vault:1.14.3
-docker run vault:1.14.3
+sudo skopeo --insecure-policy copy oci-archive:vault_<version>_amd64.rock docker-daemon:vault:<version>
```

Comment:
What do you think of adding a small comment about how to get that version? Or a command:

`yq '.version' rockcraft.yaml`

Reply:
I prefer your suggestion, as I hate having to modify the command, but if we really don't want to add the extra dependency I'd suggest at least using bash interpretation syntax so that you can set it before you copy/paste.

```
sudo skopeo --insecure-policy copy oci-archive:vault_${vault_version}_amd64.rock docker-daemon:vault:${vault_version}
```

Reply:
Done